code/index.html

```diff
@@ -2,21 +2,142 @@
+    <meta name="viewport" content="width=device-width, initial-scale=1">
   </head>
+  
+  <!-- HEADER -->
+  
   <body>
     <header>
-      <!-- Some kind of logo and navbar -->
+      <img src="img/circle.PNG" />
+      <nav>
+        <a href="#news">News</a>
+        <a href="#retreat">Retreat</a>
+        <a href="#food">Food</a>
+      </nav>
+      <div class="hamburger">
+        <div></div>
+        <div></div>
+        <div></div>
+      </div>
     </header>
 
+<!-- BIG NEWS SECTION-->
+
     <section class="big-news">
-      <!-- The big news content -->
+      <section class="container">
+        <img src="img\yogalogo.png" alt="main photo" />
+        <div>
+          <h2>
+            THIS WEEK
+          </h2>
+          <p>
+            What is going on in the world of yoga? <br> Are you keeping up with the latest trends, styles and have you heard of the latest retreats?<br><br>
+            Yoga News will take you around the world on a deep dive in everything you need to know to be an enlightened yogi 
+          </p>
+        </div>
+      </section>
     </section>
 
-    <section class="">
-      <!-- The other news  -->
+    <!-- SECTION CARDS -->
+
+    <section class="retreat" id="retreat">
+      <h3>
+        Retreats around the world
+      </h3>
+      <p>
+       Enrich your life with an unforgettable yoga retreat! Yoga News have been around the world to let you know where to book your next holiday! 
+      </p>
+      <section class="cards">
+        <section class="container">
+          <a href="https://www.yogiapproved.com/wp-content/uploads/2017/12/before-and-after-yoga-meme.jpg" target=”_blank”>
+            <h3>BALI</h3>
+            <div class="img-container">
+              <img src="img\forrest.jpg" alt="forrest yoga" />
+            </div>
+            <h4> Ever been dreaming of team yoga in the deepest forrests of Bali?
+            </h4>
+          </a>
+          <a href="https://www.yogiapproved.com/wp-content/uploads/2017/12/before-and-after-yoga-meme.jpg" target=”_blank”>
+            <h3>FLORIDA</h3>
+            <img src="img\florida.jpg" alt="beach yoga" />
+            <h4>No vacation without a beach? Florida has it all! </h4>
+          </a>
+          <a href="https://www.yogiapproved.com/wp-content/uploads/2017/12/before-and-after-yoga-meme.jpg" target=”_blank”>
+            <h3>MACAO</h3>
+            <img src="img/city1.jpg" alt="yoga in the city" />
+            <h4>Start your next yoga retreat in the  exotic urban concreate jungle of Macao</h4>
+          </a>
+          <a href="https://www.yogiapproved.com/wp-content/uploads/2017/12/before-and-after-yoga-meme.jpg" target=”_blank”>
+            <h3>INDIA</h3>
+            <img src="img/mountain1.jpg" alt="standing on a mountain" />
+            <h4>No reterat listing without India! <br>The country who gave birth to yoga! Yoga News have found the most unique retreats of India</h4>
```

Comment:
We should avoid `br` tags as much as we can - it's always better idea to split text to couple different HTML tags and use this approach to align it in different lines